src/Command/MakerCommand.php

```diff
@@ -90,13 +95,17 @@ protected function interact(InputInterface $input, OutputInterface $output): voi
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $output->write($this->linter->getLinterUserMessage());
```

Comment:
My gut reaction at this moment is: let's only show this if `$output` is in verbose mode or higher. Will be a lot of noise... for something where I think 90+% of people will be happy enough with the output.

Reply:
agree'd - unfortunately I can test for this... `$output->isVerbose()` is always true during the tests. I think this may having something to do with bridge and how we get deprecations? But im not sure.. Didn't spend a whole lot of time digging into it. 

Locally, this works as expected.. pass `-v` and we get the message...